cli/src/pcluster/config/cluster_config.py

```diff
@@ -2226,14 +2349,15 @@ def _register_validators(self):
                 checked_images.append(queue_image)
                 self._register_validator(AmiOsCompatibleValidator, os=self.image.os, image_id=queue_image)
             for compute_resource in queue.compute_resources:
-                self._register_validator(
-                    InstanceTypeBaseAMICompatibleValidator,
-                    instance_type=compute_resource.instance_type,
-                    image=queue_image,
-                )
+                for instance_type in compute_resource.instance_types:
+                    self._register_validator(
+                        InstanceTypeBaseAMICompatibleValidator,
+                        instance_type=instance_type,
+                        image=queue_image,
+                    )
                 self._register_validator(
                     InstanceArchitectureCompatibilityValidator,
-                    instance_type=compute_resource.instance_type,
+                    instance_type_info_list=list(compute_resource.instance_type_info_map.values()),
```

Comment:
is `InstanceArchitectureCompatibilityValidator` accepting a instance_type_info_list ?

Reply:
Let's keep passing `instance_type` for now (this require to add a for loop to iterate of the compute resource instance type

Reply:
nevermind, I saw that InstanceArchitectureCompatibilityValidator was modified to reflect the changes

Reply:
Yes.